queequeg.js

```diff
@@ -1,26 +1,40 @@
-var express = require('express')
-var http = require('http')
-var util = require('util')
-
-var app = express()
+var express = require('express');
+var http = require('http');
+var async = require('async');
+var Singularity = require('singularity-client');
+
+var port = process.env.PORT || 3000,
+    host = process.env.HOST || 'localhost',
+    hookPrefix = "/webook",
+    hookBaseUrl = "http://" + host + ":" + port + hookPrefix,
+    singularity_port = process.env.SINGULARITY_PORT || 7099,
+    singularity_host = process.env.SINGULARITY_HOST || host,
+    singularity_base = process.env.SINGULARITY_BASE || '/singularity';
+
+var singularity = new Singularity({
+    singularity: {
+      baseUrl: "http://" + singularity_host + ":" + singularity_port + singularity_base
```

Comment:
the util.format style keeps the actual shape of the URL and inserts the values which is more readable.